packages/ckeditor5-dev-env/lib/release-tools/utils/getpackagestorelease.js

```diff
@@ -29,24 +32,43 @@ module.exports = function getPackagesToRelease( pathsToPackages ) {
 	function filterPackagesToRelease( repositoryPath ) {
 		process.chdir( repositoryPath );
 
-		const gitVersion = versionUtils.getLastTagFromGit();
-		const changelogVersion = versionUtils.getLastFromChangelog();
 		const packageJson = getPackageJson();
 		const repositoryName = packageJson.name;
 
+		const newVersion = findVersionInChangelog( options.changes, repositoryName ) || options.version;
+
 		// If these versions aren't equal, it means that the package is ready to release
-		// because we assume that a version from changelog is the latest.
-		if ( gitVersion !== changelogVersion ) {
+		// because we assume that a version from `package.json` is the latest.
+		if ( packageJson.version !== newVersion ) {
 			packagesToRelease.set( repositoryName, {
 				previousVersion: packageJson.version,
-				version: changelogVersion
+				version: newVersion
 			} );
 		}
 
 		return Promise.resolve();
 	}
 };
 
+// Search for a new version for specified package in the main changelog description.
+// It must be defined as following:
+//
+//      [packageName](https://www.npmjs.com/package/packageName): v0.0.1 => [v1.0.0](https://github.com/.../releases/tag/v1.0.0)
```

Comment:
TODO: Check whether it works with the following entries:

```md
* [@ckeditor/ckeditor5-word-count](https://www.npmjs.com/package/@ckeditor/ckeditor5-word-count): v0.0.2
* [@ckeditor/ckeditor5-build-balloon](https://www.npmjs.com/package/@ckeditor/ckeditor5-build-balloon): v19.0.0 => v19.0.1
```

Reply:
It won't Require changes. For upgrading existing repo:

```js
\[@ckeditor\/ckeditor5-word-count\].*: v[\d.]+ \=\> v([\d.]+)
```

For new release:

```js
\[@ckeditor\/ckeditor5-word-count\].*: v([\d.]+)
```

The order of execution is important. First must be checked an option with already existing package.

Reply:
Done.